### PR TITLE
Bump docker to version 10.1.2

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
 FROM postgres:13.3
```
Comment (hanefi) [inline Dockerfile:3]:
```suggestion
FROM postgres:13.4
```

File: alpine/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:13.2-alpine
```
Comment (hanefi) [inline alpine/Dockerfile:3]:
```suggestion
FROM postgres:13.4-alpine
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+### citus-docker v10.1.2.docker (August 18,2021) ###
+
+* Bump Citus version to 10.1.2
```
Comment (hanefi) [inline CHANGELOG.md:3]:
```suggestion
* Bump Citus version to 10.1.2
* Bump PG versions to 13.4 and 12.8
```

File: postgres-12/Dockerfile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/postgres-12/postgres-12.tmpl.dockerfile.
 FROM postgres:12.7
```
Comment (hanefi) [inline postgres-12/Dockerfile:3]:
```suggestion
FROM postgres:12.8
```

**Review by hanefi — COMMENTED** [review]
There are new PG versions available as packages and docker images.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+* Bump PG versions to 13.4 and 12.8
```
Comment (hanefi) [inline CHANGELOG.md:4]:
```suggestion

* Bump PG versions to 13.4 and 12.8
```

Sorry for misleading you. Making an edit on the web browser is tricky and I missed an empty line on my previous suggestion.

File: alpine/Dockerfile (hunk 2 above)
Comment (gurkanindibay) [reply alpine/Dockerfile:3]: Rollbacked due to travis error as below
https://app.travis-ci.com/github/citusdata/docker/jobs/532229232

File: postgres-12/Dockerfile (hunk 4 above)
Comment (gurkanindibay) [reply postgres-12/Dockerfile]: Fixed

**Review by hanefi — APPROVED** [review]

**Pull request merged**